### DISCUSSION TITLE
The reason to name download/upload artifact on GitHub

### DISCUSSION THREAD

**Discussion by abitrolly** [pypa/cibuildwheel#1158] [Ideas]
This.

![image](https://user-images.githubusercontent.com/8781107/174871616-3251dfe4-d12f-4ade-95a2-602219f2a43c.png)

**Comment by henryiii** [comment]
It is the build artifact? It's got wheels and an SDist in it. Seems fine (and consistent) to me if you put everything in one artifact (which is easiest most of the time). It's always going to be under "artifacts" regardless of the name you pick.

**Reply by abitrolly** [reply]
Yes. It is what minimal example produces. It will be more user friendly to name it `wheels-` at least for examples that don't do anything else with those uploads https://github.com/pypa/cibuildwheel/pull/1159

**Reply by henryiii** [reply]
But it's not just wheels. It's the SDist too.

**Reply by abitrolly** [reply]
Not for https://github.com/pypa/cibuildwheel/blob/main/examples/github-minimal.yml
